modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/SeriesEndpoint.java

```diff
@@ -239,6 +243,12 @@ protected void activate(ComponentContext cc) {
       logger.debug("Configured server url is {}", ccServerUrl);
       if (ccServerUrl != null)
         this.serverUrl = ccServerUrl;
+
+      Dictionary dict = cc.getProperties();
+      List<String> keys = Collections.list(dict.keys());
+      Map<String, Object> dictCopy = keys.stream()
+              .collect(Collectors.toMap(Function.identity(), dict::get));
+      modified(dictCopy);
```

Comment:
You can also add the properties to the activate method as a parameter, then you don't need to transform the dictionary to a map. :) Both activate() and modified() can take multiple parameters, e.g. component context, properties or bundle context.